fix drop folder file conflict

yogetter opened the pull request:
In version 8.2.1, owcloud use new method to check file conflict in js
But if using drop folder to upload, new method did not check files relativepath, this will casue file conflict error.

yogetter committed:
fix drop folder file conflict

Wilson-Su commented on apps/files/js/file-upload.js:
```diff
@@ -210,7 +210,16 @@ OC.Upload = {
+			//avoid drop folder conflict error
+			//check relativepath before check conflict
+                        var relativePath = upload.files[0].relativePath;
+                        if (relativePath == "" ||  relativePath == null){
```
Yous should use three-equals (===) instead of two-equals (==)

Wilson-Su commented on apps/files/js/file-upload.js:
```diff
@@ -210,7 +210,16 @@ OC.Upload = {
+			//check relativepath before check conflict
+                        var relativePath = upload.files[0].relativePath;
+                        if (relativePath == "" ||  relativePath == null){
+                                relativePath = true;
```
Please use single-quote (') instead of double-quote (")
It would be better to not use if. (variable1 = variable1 === '' || variable1 === null || variable1 === undefined)

yogetter committed:
Fix drop folder file conflict
In 8.2.1, use js to check file conflict,
but will occur error when drop folder.

yogetter committed:
Fix drop folder file conflict
In 8.2.1, use js to check file conflict,
but will occur error when drop folder.

using ternary operator to replace if else
using three-equals to replace two-equals

yogetter commented:
using ternary operator to replace if else
using three-equals to replace two-equals

yogetter commented:
fix three-equals

yogetter committed:
Fix drop folder file conflict
In 8.2.1, use js to check file conflict,
but will occur error when drop folder.

using ternary operator to replace if else
using three-equals to replace two-equals
add check undefined

yogetter committed:
Fix drop folder file conflict
In 8.2.1, use js to check file conflict,
but will occur error when drop folder.

using ternary operator to replace if else
using three-equals to replace two-equals
add check undefined
